installer/InstallBuilder/linuxdpkg.py

```diff
@@ -166,9 +166,11 @@ def BuildPackage(self):
             return
 
         # Build the package - 'cd' to the directory where we want to store result
-        dpkg_path = 'dpkg'
         if "DPKG_LOCATION" in self.variables:
             dpkg_path = self.variables["DPKG_LOCATION"]
+        # Use the system's version of dpkg-deb if present.
+        if os.path.exists('/usr/bin/dpkg-deb'):
+            dpkg_path = '/usr/bin/dpkg-deb'
```

Comment:
I don't think that any of this is correct. I control this from one level up (in the makefiles). In the OMS makefile, I have:

```
ifeq ("$(wildcard /usr/bin/dpkg-deb)","")
    DPKG_LOCATION="--DPKG_LOCATION=$(PAL_DIR)/installer/InstallBuilder/tools/bin/dpkg-deb-$(PF_ARCH)"
else
    DPKG_LOCATION=
endif
```

Then, when building DPKG kits, I'll include `$(DPKG_LOCATION)` on the installbuilder line. This will override (or not override) what dpkg program to use.  Finally, if one of the installbuilder data files has:

```
PACKAGE_TYPE: 'DPKG'
```

This tells installbuilder to build a DPKG package, and by default it will use the system location UNLESS I've overridden this.

So, all that said and done, I don't think that you should have this change at all.